reports/data_helper.py

```diff
@@ -0,0 +1,31 @@
+import pandas as pd
+
+
+def prepare_expense_data(transactions_by_category):
+    expense_data = []
```

Comment:
Add docstrings, please

Reply:
And type annotations through the code changes

Reply:
Added docstrings to all files and type annotations.